setup.cfg

```diff
@@ -68,11 +68,13 @@ console_scripts =
 [flake8]
 # E302: Expected 2 blank lines
 # E501: Line too long
+# F401: imported but unused. pyright already checks this.
 # F403: 'from module import *' used; unable to detect undefined names
 # F405: Name may be undefined, or defined from star imports
 # W503: Line break occurred before a binary operator
-ignore = E302, E501, F403, F405, W503
+ignore = E302, E501, F401, F403, F405, W503
 exclude = docs, .venv
+max-line-length = 127
```

Comment:
Can we get rid of this? It seems like it's redundant with `E501`.

Reply:
Will remove it. I moved it from the `Makefile` to the configuration file, originally it's from [here](https://github.com/rstudio/py-shiny/blob/main/Makefile#L61).